### ISSUE TITLE
False positives when using .run() method

### ISSUE THREAD

**Issue by keirlawson** [Gillespie59/eslint-plugin-angular#137]
The plugin is giving errors (`angular/ng_module_getter`, `angular/ng_di`) on the following line of my code:

```js
mocha.run();
```

Presumably because it has misidentified the `run()` call as an angular run block.

**Comment by Gillespie59** [comment]
This issue has been solved. And available in the 0.2.1 release. 

Thanks @keirlawson
